plugins/org.eclipse.sirius.diagram.ui/src-diag/org/eclipse/sirius/diagram/ui/tools/internal/editor/DOperationHistoryListener.java

```diff
@@ -79,6 +79,9 @@ protected boolean eventShouldTriggerArrange(OperationHistoryEvent event) {
             } else if (Messages.SiriusContainerEditPolicy_arrangeCommandLabel.equals(originalCommand.getLabel())) {
                 // The operation is already an arrange command. No need to launch another one.
                 result = false;
+            } else if (Messages.PasteStyleDataCommand_label.equals(originalCommand.getLabel())) {
+                // Paste style operation doesn't need to launch an arrange command. 
+                result = false;
```

Comment:
Does it have anything to do with the "Arrange Linked Border Nodes" on D&D or is it just another fix?